addressableled/robot.py

```diff
@@ -0,0 +1,50 @@
+from wpilib import run, TimedRobot, AddressableLED
+
+kLEDBuffer = 60
+
+class MyRobot(TimedRobot):
+    def robotInit(self):
+        # PWM Port 9
+        # Must be a PWM header, not MXP or DIO
+        self.m_led = AddressableLED(9)
```

Comment:
Our examples don't prefix member variables with `m_`, as the mandatory `self.` already implies it's a member variable.

Reply:
okay!